.gitignore

```diff
@@ -114,3 +114,4 @@ env/
 
 ### Configuration files ###
 .env
```

Comment:
Now that you've added poetry, you can actually move normal running of your app (non-vagrant) to Poetry as well.

You can delete requirements.txt and the setup.sh script. You should then update your README to match.

If you add this file to your project, then it will create a virtual env folder when you run `poetry install`: 
https://github.com/CorndelWithSoftwire/DevOps-Course-Starter/blob/master/poetry.toml 
At which point you should change this `.env` line in .gitignore to the newly created .venv.